packages/react/src/components/Button/Button.js

```diff
@@ -38,6 +38,8 @@ const Button = React.forwardRef(function Button(
     [`${prefix}--btn`]: true,
     [`${prefix}--btn--field`]: size === 'field',
     [`${prefix}--btn--sm`]: size === 'small' || small,
+    [`${prefix}--btn--lg`]: size === 'large',
```

Comment:
Wondering if we should keep this size as `lg` to be consistent with other the other input size conventions, and we can make a note to change `small` to `sm` in our next major release.